code/modules/status_system/statusSystem.dm

```diff
@@ -689,25 +690,65 @@ var/global/list/statusGroupLimits = list("Food"=4)
 
 				if(2)
 					damage_tox = (2 * prot)
-					damage_brute = (2 * prot)
+					damage_brute = (1 * prot)
```

Comment:
please remove this dumb copied code every time in the switch statement, this is just tox = stage*prot, brute = (stage/2)*prot